streams/ru.m3u

```diff
@@ -211,187 +197,111 @@ https://uiptv.do.am/1ufc/602079679/playlist.m3u8
-#EXTINF:-1 tvg-id="BRIDGEHits.ru",BRIDGE Hits
-http://31.148.48.15/Bridge_TV_Dance/index.m3u8
-#EXTINF:-1 tvg-id="BRIDGEClassic.ru",BRIDGE Classic
-http://31.148.48.15/Bridge_TV_Classic/index.m3u8
 #EXTINF:-1 tvg-id="RTGTV.ru",RTG TV
 http://194.143.148.28:8080/RTG_HD/index.m3u8
-#EXTINF:-1 tvg-id="EGETV.ru",EGE TV
+#EXTINF:-1 tvg-id="EGETV.ru",ЕГЭ ТВ
 http://194.143.148.28:8080/RT_DOC/index.m3u8
-#EXTINF:-1 tvg-id="YuTV.ru",YuTV
-https://livetv.mylifeisgood.net.ru/channels/u_ott_tv_hq.m3u8
-#EXTINF:-1 tvg-id="India.ru",India
-http://31.148.48.15/Zee_TV_Russia/index.m3u8
-#EXTINF:-1 tvg-id="Evrokino.ru@SD",Evrokino
-http://31.148.48.15/Evrokino/index.m3u8
-#EXTINF:-1 tvg-id="IllusionPlus.ru@SD",Illusion+
-http://31.148.48.15/Illusion_plus/index.m3u8
-#EXTINF:-1 tvg-id="Mama.ru",Mama
+#EXTINF:-1 tvg-id="Mama.ru",Мама
 http://stv.mediacdn.ru/live/cdn/mama/playlist.m3u8
-#EXTINF:-1 tvg-id="Multimuzyka.ru",Mult i muzyka
+#EXTINF:-1 tvg-id="Multimuzyka.ru",Мульт и музыка
 http://stv.mediacdn.ru/live/cdn/multimuzika/playlist.m3u8
-#EXTINF:-1 tvg-id="Mult.ru",Mult
+#EXTINF:-1 tvg-id="Mult.ru",Мульт
 http://stv.mediacdn.ru/live/cdn/mult/playlist.m3u8
-#EXTINF:-1 tvg-id="SuperGeroi.ru",SuperGeroi
+#EXTINF:-1 tvg-id="SuperGeroi.ru",СуперГерои
 http://178.124.179.122:8088/Malish/index.m3u8
 #EXTINF:-1 tvg-id="Cinema.ru",Cinema
 http://catchup.videoline.ru/cinema/tracks-v1a1/mono.m3u8
-#EXTINF:-1 tvg-id="ZhivayaPlaneta.ru@SD",Zhivaya Planeta
-http://31.148.48.15/Zhivaya_Planeta/index.m3u8
-#EXTINF:-1 tvg-id="HDL.ru",HDL
-http://31.148.48.15/HD_Life/index.m3u8
-#EXTINF:-1 tvg-id="AutoPlus.ru@SD",Auto Plus
-http://31.148.48.15/Avto_Plus/index.m3u8
 #EXTINF:-1 tvg-id="Bolt.ru",Bolt
 http://95.142.88.250:80/Bolt/index.m3u8
-#EXTINF:-1 tvg-id="Kineko.ru",Kineko
+#EXTINF:-1 tvg-id="Kineko.ru",Кинеко
 http://194.143.148.28:8080/KINEKO/index.m3u8
-#EXTINF:-1 tvg-id="TERRA.ru",TERRA
-https://livetv.mylifeisgood.net.ru/channels/terra.m3u8
 #EXTINF:-1 tvg-id="scifi.ru@SD",.sci-fi
 http://178.124.179.122:8088/Sci-Fi/index.m3u8
-#EXTINF:-1 tvg-id="Solnce.ru",Solnce
+#EXTINF:-1 tvg-id="Solnce.ru",Солнце
 http://tv.mediacdn.ru/live/solntse/playlist.m3u8
-#EXTINF:-1 tvg-id="TNT4International.ru@SD",TNT4 International
+#EXTINF:-1 tvg-id="TNT4International.ru@SD",ТНТ4 International
 https://cdn4.skygo.mn/live/disk1/TNT4/HLSv3-FTA/TNT4.m3u8
 #EXTINF:-1 tvg-id="KHLPrime.ru",KHL Prime
 http://194.143.148.28:8080/KHL_HD/index.m3u8
-#EXTINF:-1 tvg-id="Konniymir.ru",Konniy mir
+#EXTINF:-1 tvg-id="Konniymir.ru",Конный Мир
 https://cdn-01.bonus-tv.ru/kmir/playlist.m3u8
-#EXTINF:-1 tvg-id="SmaylikTV.ru@SD",Smaylik TV
+#EXTINF:-1 tvg-id="SmaylikTV.ru@SD",Смайлик TV
 https://5f05e059699ae.streamlock.net/WEB_Smilik/ngrp:Smilik.stream-adaptive/playlist.m3u8
-#EXTINF:-1 tvg-id="MoyaPlaneta.ru",Moya Planeta
+#EXTINF:-1 tvg-id="MoyaPlaneta.ru",Моя Планета
 http://194.143.148.28:8080/MoyaPlanetaHD/index.m3u8
-#EXTINF:-1 tvg-id="STSkids.ru",STS kids
-http://31.148.48.15/STS_Kids_HD/index.m3u8
```

Comment:
http://178.217.72.66:8080/STS/index.m3u8 If by any chance this isn't in the repo, that can be switched up.